supereeg/model.py

```diff
@@ -312,50 +328,27 @@ def predict(self, bo, nearest_neighbor=False, match_threshold='auto',
             mo = self
 
         #blur out model to include brain object locations
-        mo.set_locs(bor.get_locs(), include_original_locs=True)
+        mo.set_locs(bor.get_locs(), include_original_locs=include_original_locs)
```

Comment:
What happens when `include_original_locs == False` here?  Would we then round the observed electrode locations to the nearest locations in `mo`?

Reply:
no.. it fails when the brain object contains model locations currently. investigating